assets/large_language_models/rag/components/src/flow_creation.py

```diff
@@ -341,15 +341,7 @@ def main(args, ws, current_run, activity_logger: Logger):
                 file.write(codecs.decode(_MODIFY_PROMPT, 'unicode_escape'))
 
         # upload code
-        try:
-            details = current_run.get_details()
-            user_name = details.get("submittedBy", "systemcreated")
-            user_name = user_name.lower().replace(" ", "")
-        except Exception:
-            # For local runs, get_details fails
-            user_name = "systemcreated"
-
-        yaml_path = upload_code_files(ws, user_name) + "/flow.dag.yaml"
+        yaml_path = upload_code_files(ws) + "/flow.dag.yaml"
```

Comment:
Not an issue. Just wondering what 'dag' stands for?

Reply:
directed acyclic graph - this is convention set by prompt flow team.

Reply:
Directed Acyclic Graph!